scratch/evaluate_gaming_traffic.cc

```diff
@@ -37,19 +37,26 @@ Ptr<PacketSink> packetSink;    /* Pointer to the packet sink application */
 uint64_t lastTotalRx = 0;      /* The value of the last total received bytes */
 Time lastPacketTime = Seconds (-1.0);
 
+/** Simulation Arguments **/
+bool csv = false;      /* Enable CSV output. */
+bool summary = true;  /* print application layer traffic summary */
+double simulationTime = 10.0;  /* simulation time in seconds */
+std::string game = "ns3::CrazyTaxiStreamingServer"; /* TypeId of the game */
```

Comment:
comments should either have a fixed space from `;` (1 or 2 spaces) or be all aligned